db/data/20201008050453_migrate_permissions_to_accesses.rb

```diff
@@ -0,0 +1,26 @@
+class MigratePermissionsToAccesses < ActiveRecord::Migration[5.2]
+  require "tasks/scripts/create_accesses_from_permissions"
+
+  def up
+    # This is a dev-env only migration
+    unless Rails.env.development?
+      return
+    end
+
+    Organization.all.each do |organization|
+      CreateAccessesFromPermissions.do(organization: organization, dryrun: false)
+    end
+
+    # Promoting admins with custom permissions or missing accesses to power_users
+    custom_admins = User.admins.select { |admin| (!admin.power_user? && admin.accesses.empty?) }
+
+    custom_admins.each do |custom_admin|
```

Comment:
This is a catchall to make any left over admins power-users. This is necessary since the `CreateAccessesFromPermissions` script skips admins with custom permissions, and should be safe since this migration only runs in dev.